about.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script src="script.js" defer></script>
+    <title>Estudio Polanco</title>
+  </head>
+  <body>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn">&times;</a>
+      <div class="overlay-content">
+        <a href="#work" id="mybtn">Program</a>
+        <a href="#about">About</a>
+        <a href="#contact">Gallery</a>
+      </div>
+    </div>
+    <div class="topbar">
+      <ul>
+        <li>
+          <a href="">
+            <img class="topbar-icon" src="./img/fb.svg" alt="Fb logo" />
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <img
+              class="topbar-icon"
+              src="./img/Twitter.svg"
+              alt="Twitter logo"
+            />
+          </a>
+        </li>
+        <li>Logout</li>
+        <li>Mypage</li>
+        <li>Korean</li>
+      </ul>
+    </div>
+    <nav>
+      <div class="image-nav">
+        <img src="./img/logo-footer.png" alt="Image Logo" />
```

Comment:
- [ ] **OPTIONAL**
Please make the logo navigates to the home page with a link `<a>`.

_Please notice that this a high-recommended to fix requirement, and I'm sorry it didn't show up earlier 🙏 😄_